code/controllers/configuration/entries/game_options.dm

```diff
@@ -235,6 +235,9 @@
 /datum/config_entry/string/alert_red_downto
 	config_entry_value = "The station's destruction has been averted. There is still however an immediate serious threat to the station. Security may have weapons unholstered at all times, random searches are allowed and advised."
 
+/datum/config_entry/string/alert_omega
+	config_entry_value = "Attention, Code Omega has been initiated. Consider all employment contracts terminated, effective immediately."
```

Comment:
We don't need to bloat the config even more.
Just make it static string

Reply:
Actually make this alert message be able to be set by admin upon trying to change the alert to this and keep this as default value or wharever